### DISCUSSION TITLE
Improve notifications

### DISCUSSION THREAD

**Discussion by Tardo** [Tecnativa/docker-duplicity#731] [Ideas]
apprise: https://github.com/caronc/apprise
Integrating this tool opens up the possibility of sending a notification to a lot of services in a simple way.

What do you think about integrating it to send error and/or confirmation notifications?

**Comment by pedrobaeza** [comment]
It can be interesting, but more than focusing on this project itself, I think it may be at Doodba level to get from any of the possible parts, don't you think?

**Comment by Tardo** [comment]
Sounds good to me... so, we need use: https://github.com/caronc/apprise-api
